terminatorlib/keybindings.py

```diff
@@ -91,7 +95,7 @@ def reload(self):
                     self._lookup[mask][keyval] = action
                     self._masks |= mask
 
-    def _parsebinding(self, binding):
```

Comment:
Why change this method name?

Reply:
The underscore kept it as an internal method. This affected the tests from being able to access the method.